etcd/assets/service_checks.json

```diff
@@ -28,6 +28,21 @@
             "url"
         ],
         "name": "Healthy",
-        "description": "Returns `CRITICAL` when a member is unhealthy"
+        "description": "Returns `CRITICAL` when a member is unhealthy. This service check is only available on the legacy version of the etcd check."
+    },
+    {
+        "agent_version": "6.8.0",
```

Comment:
The Prometheus-based check was initially added in [6.8.0](https://github.com/DataDog/integrations-core/blob/master/etcd/CHANGELOG.md#160--2018-11-07--agent-680)